src/Client.php

```diff
@@ -244,4 +251,11 @@ public function getLogger(): \Psr\Log\LoggerInterface
 
         return $this->logger;
     }
+
+    public function enableDebugLogging(bool $enable = true): Client
```

Comment:
I'd rather pollute a config array with "extra" options which the user doesn't usually need to be concerned with than pollute the constructor signature - if we have to add more options like this, I'd prefer to see those as options in a config/options bag than keep adding optional parameters to the constructor.

If the config is only supposed to be for connection details, then it should probably have a more specific name than `$config` (`$connectionConfig`?) and then we have a separate parameter for an options bag - otherwise I feel like `$config` is the perfect place for an "extra" option to go.